go.mod

```diff
@@ -158,14 +164,60 @@ require (
 	gopkg.in/yaml.v2 v2.2.2
 	istio.io/api v0.0.0-20190913194855-8920ca438254
 	istio.io/gogo-genproto v0.0.0-20190731221249-06e20ada0df2
-	istio.io/operator v0.0.0-20190918001506-3a5dede6e691
+	istio.io/operator v0.0.0-20190923170413-6e6db1228c1f
 	istio.io/pkg v0.0.0-20190905225920-6d0bbfe3b229
-	k8s.io/api v0.0.0-20190222213804-5cb15d344471
-	k8s.io/apiextensions-apiserver v0.0.0-20190221221350-bfb440be4b87
-	k8s.io/apimachinery v0.0.0-20190221213512-86fb29eff628
-	k8s.io/cli-runtime v0.0.0-20190221101700-11047e25a94a
-	k8s.io/client-go v10.0.0+incompatible
-	k8s.io/helm v2.13.1+incompatible
+	k8s.io/api v0.0.0
+	k8s.io/apiextensions-apiserver v0.0.0
+	k8s.io/apimachinery v0.0.0
+	k8s.io/cli-runtime v0.0.0
+	k8s.io/client-go v11.0.1-0.20190409021438-1a26190bd76a+incompatible
```

Comment:
According to https://github.com/kubernetes/client-go, client-go 10.0 is best fit for kubernetes 1.14. Why upgrade client-go singly?

Reply:
our kubernetes dependency is 1.15. but we still support older kubernetes versions, just using newer kubernetes library that are backwards compatible.

All of there versions are pinned at kubernetes 1.15

Reply:
also this is what is used in operator, so unless we remove istioctl from this repo (which I think we should) we have to match them